src/firmware_new/boards/frankie_v1/main.c

```diff
@@ -56,7 +56,11 @@ UART_HandleTypeDef huart3;
 
 PCD_HandleTypeDef hpcd_USB_OTG_FS;
 
+/* Definitions for defaultTask */
 osThreadId_t defaultTaskHandle;
+const osThreadAttr_t defaultTask_attributes = {.name     = "defaultTask",
+                                               .priority = (osPriority_t)osPriorityNormal,
+                                               .stack_size = 1024 * 4};
```

Comment:
Any reason for choosing this size? Just asking because the previous stack side being 1024 only.

Reply:
this is autogenerated, 1024 is how many words we we allocate for the stack, in the new version of cube stack_size is now in bytes, which is where the 1024 *4 is from

since we upgraded, cube autogenerates 1024*4 instead of 1024